librz/core/tui/vmenus_graph.c

```diff
@@ -242,11 +242,27 @@ RZ_IPI int __core_visual_view_graph_update(RzCore *core, RzCoreVisualViewGraph *
 	rz_cons_strcat_at(mainColstr, colx, 2, colw * 2, colh);
 	rz_cons_strcat_at(refsColstr, colx * 2, 2, colw, colh);
 
-	char *output = rz_core_cmd_strf(core, "pd %d @e:asm.flags=0 @ 0x%08" PFMT64x "; pds 256 @ 0x%08" PFMT64x "\n",
-		32, status->addr, status->addr);
+	RzConfigHold *hc = rz_config_hold_new(core->config);
+	if (!hc) {
+		return 0;
+	}
+	rz_config_hold_i(hc, "asm.flags", NULL);
+	rz_config_set_i(core->config, "asm.flags", 0);
+	char *output1 = rz_core_print_cons_disassembly(core, status->addr, 32, 0);
+	rz_config_hold_restore(hc);
+	rz_config_hold_free(hc);
+
+	ut64 oldoff = core->offset;
+	rz_core_seek(core, status->addr, true);
+	char *output2 = rz_core_print_disasm_strings(core, RZ_CORE_DISASM_STRINGS_MODE_BYTES, 256, NULL);
+	rz_core_seek(core, oldoff, true);
+
+	char *output = rz_str_concat(output1, output2);
```

Comment:
Is `rz_str_concat` really needed? Can't you use `output1 = rz_str_append(output1, output2)`?

Reply:
I missed that one function, thanks